docs/src/main/paradox/index.md

```diff
@@ -1,70 +1,85 @@
 
-We will create a main class from which we create an `akka.actor.ActorSystem` (think of this as a container in which your Actors run). Once we have created an `ActorSystem` we create a couple of Actors; 3 instances of a `Greeter` Actor and 1 instance of a `Printer` Actor. We will thereafter send messages to the `Greeter` Actor instances which will keep these messages internally. Finally, we send instruction messages to our `Greeter` Actors that will trigger them to send messages to the `Printer` Actor for output.
+The output should look _something_ like this (scroll all the way to the right to see the Actor output):
+ 
+```
+[info] Compiling 1 Scala source and 1 Java source to /Users/x/akka-quickstart-scala/target/scala-2.12/classes...
+[info] Running com.lightbend.akka.sample.AkkaQuickstart
+>>> Press ENTER to exit <<<
+[INFO] [05/09/2017 09:57:15.979] [helloAkka-akka.actor.default-dispatcher-2] [akka://helloAkka/user/printerActor] Greeting received (from Actor[akka://helloAkka/user/howdyGreeter#-1854995773]): Howdy, Akka
+[INFO] [05/09/2017 09:57:15.980] [helloAkka-akka.actor.default-dispatcher-2] [akka://helloAkka/user/printerActor] Greeting received (from Actor[akka://helloAkka/user/helloGreeter#-1072877049]): Hello, Scala
+[INFO] [05/09/2017 09:57:15.980] [helloAkka-akka.actor.default-dispatcher-2] [akka://helloAkka/user/printerActor] Greeting received (from Actor[akka://helloAkka/user/goodDayGreeter#1972065097]): Good day, Play
+[INFO] [05/09/2017 09:57:15.980] [helloAkka-akka.actor.default-dispatcher-2] [akka://helloAkka/user/printerActor] Greeting received (from Actor[akka://helloAkka/user/howdyGreeter#-1854995773]): Howdy, Lightbend
+[success] Total time: 5 s, completed May 9, 2017 9:52:34 AM
+```
+   
+Congratulations, you just ran your first Akka app. Now take a look at what happened under the covers. 
 
-Let's take a look at the architecture. As said above, we will run everything from the main class. The main class will create and interact with the Actors:
+## What Hello World does
+
+As you saw in the console output, the example outputs several greetings. Let’s take a look what happens at runtime.
 
 ![Architecture](images/hello-akka-architecture.png)
 
-Once we have created the Actors we will send messages to them:
+First, the `main` class creates an `akka.actor.ActorSystem`, a container in which Actors run. Next, it creates three instances of a Greeter Actor and one instance of a Printer Actor. 
 
-![Messages](images/hello-akka-messages.png)
+The example then sends messages to the Greeter Actor instances, which store them internally. Finally, instruction messages to the Greeter Actors trigger them to send messages to the Printer Actor, which outputs them to the console:
 
-Don't worry if this doesn't make complete sense at this point! This tutorial will describe all the core concepts you need to build an Akka application.
+![Messages](images/hello-akka-messages.png)
 
-## The 3 Minute Exploration
```

Comment:
Running it is moved to the very beginning (and is not optional).

"It is important for newbies to try something quickly. So, instead of allowing the user to choose when to run the sample, I moved the instructions for downloading and running right up front. I think it provides a better experience."

I think that is a good move. That was my original purpose with this guide.